broker-daemon/models/block-order.js

```diff
@@ -476,6 +476,100 @@ class BlockOrder {
 
     return BlockOrder.fromStorage(blockOrderId, value)
   }
+
+  /**
+   * Grabs all orders for a range of block orders
+   * @param {sublevel} store
+   * @param {string} first
+   * @param {string} last
+   * @returns {void}
+   */
+  static async getOrdersForRange (store, first, last) {
+    return getRecords(
+      store,
+      (key, value) => {
+        const { order, state, error, dates } = JSON.parse(value)
+        return { order: Order.fromObject(key, order), state, error, dates }
+      },
+      // limit the orders we retrieve to those that belong to this blockOrder, i.e. those that are in
+      // its prefix range.
+      Order.rangeForIds(first, last)
+    )
+  }
+
+  static async getFillsForRange (store, first, last) {
```

Comment:
Seems odd to have this defined as a static on the BlockOrder. Any reason why you did that?

Reply:
@treygriffith I didnt have a strong reason to keep them on the BlockOrder. I only kept the methods here because of `populateOrders` and `populateFills`.

I debated moving this to Fill/Order respectively